preupg/xml_manager.py

```diff
@@ -201,19 +201,16 @@ def get_updated_text(self, solution_text, text, line):
                 updated_text = line.replace(solution_text + "_TEXT",
                                             "").replace(solution_text + "_HTML",
                                                         "")
-
         else:
             updated_text = text
         return updated_text
 
-    def update_html(self, result_name, solution_txt_dirnames, extension="html"):
-        """
-         Function updates a XML or HTML file with relevant solution
-         texts
+    def update_report(self, report_filename):
+        """Update either XML or HTML report with relevant solution texts.
```

Comment:
Pete, it's according to the [PEP 257: docstring conventions](https://www.python.org/dev/peps/pep-0257/).

Reply:
hmm.. not sure. I really don't see this. Correct form according to PEP is e.g.:
```python
def func()
    """
    One-line string.
    """
```
but I really don't see use any example for one-line doc there. (maybe it's described in text, but I didn't read it now). Multiline would be correct. Maybe there is even change in PEP documentation sice time I checked it earlier, because there was recommended use simple quotes instead of triple-quotes for one-liners.

Reply:
> One-liners are for really obvious cases. They should really fit on one line. For example:
> ```
> def kos_root():
>   """Return the pathname of the KOS root directory."""
> ```

They advocate trimmed multiline docstrings:
> ```
> def foo():
>   """A multi-line
>   docstring.
>   """
> ```

I did put the triple quotes on the new line because of the 80 chars line rule - I know that many are against this one, but I still adhere to it (and I think it's quite a subjective matter, i.e. prone to the bikeshed effect).